package NFillFactors;

message TSource {
    required string ClassName = 1;
};

message TFillFactorsCodegenOptions {
    required string MethodName = 1;
    repeated string Includes = 2;
    repeated string Tags = 3;
    repeated string ForbiddenTags = 8;
    repeated TSource Sources = 4;
    optional string FactorEnumNamespace = 5;
    required string Slice = 6;
    optional string NestedNamespace = 7;
}
